cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/command/GradleCycloneDxGenerateCommand.java

```diff
@@ -44,6 +44,10 @@ public class GradleCycloneDxGenerateCommand extends AbstractGradleGenerateComman
     protected Path doGenerate(String buildCmdOptions) {
         ProcessBuilder processBuilder = new ProcessBuilder().inheritIO();
 
+        // Making the plugin version configurable
+        processBuilder.command()
+                .add(String.format("PLUGIN_VERSION=%s", toolVersion()));
```

Comment:
Didn't you want `environment()` here?

Reply:
this is the Gradle CycloneDX plugin version, which is specified in:
```      
gradle-cyclonedx:
        default-version: "1.7.4"
```
or using the `--tool-version` parameter.

Reply:
I think my comment was more about using the `command()` method. This adds the passed parameter to the **command** that is being executed, but from what I see you want to set an environment variable, for which I would expect using `environment().put("PLUGIN_VERSION", toolVersion())`.

Reply:
ok right, I was going for a simpler way, to execute the command like `PLUGIN_VERSION=1.7.4 gradle etc etc `

Reply:
Aaaah. I wasn't sure if this was possible with `ProcessBuilder`. If it is, I'm fine with it!

Reply:
right, you are correct, let's not abuse the command and use the proper way!